src/sentry/constants.py

```diff
@@ -123,9 +123,10 @@ def get_all_languages():
 DEFAULT_SORT_OPTION = 'date'
 
 # Setup languages for only available locales
-LANGUAGE_MAP = dict(settings.LANGUAGES)
```

Comment:
Why is this gone?

Reply:
I was looking through unused constants and removed things that I didn't see used in sentry, getsentry and sentry-plugins